src/calculateRentalCost.js

```diff
@@ -1,10 +1,23 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 0;
+
+  if (days < 3) {
+    rentalCost = days * 40;
+  } else if (days < 7) {
+    rentalCost = days * 40 - 20;
+  } else {
+    rentalCost = days * 40 - 50;
+  }
+
+  return rentalCost;
 }
```

Comment:
Consider using constants for the daily rental cost, and the discount values. This will make the code more readable and maintainable. For example, use a constant like `const DAILY_RENTAL_COST = 40;` and then use it in your calculations.